include/wx/msw/wrapwin.h

```diff
@@ -30,7 +30,7 @@
 
 // For IPv6 support, we must include winsock2.h before winsock.h, and
 // windows.h include winsock.h so do it before including it
-#if wxUSE_IPV6
+#if defined(wxUSE_IPV6) || defined(wxUSE_WINSOCK2)
```

Comment:
Sorry, this is wrong. `wxUSE_XXX` must always be defined as 0 or 1, rather than being defined or not. So this should be
```suggestion
#if wxUSE_IPV6 || wxUSE_WINSOCK2
```
but it would be even better to do something like
```cpp
/* Using IPv6 requires winsock2.h under MSW. */
#if wxUSE_IPV6
#    undef wxUSE_WINSOCK2
#    define wxUSE_WINSOCK2 1
#endif
```
in `include/wx/msw/chkconf.h`.

Reply:
> Sorry, I forgot to mention that include/wx/chkconf.h also needs to be updated (I should have just pointed you to docs/contributing/how-to-add-new-wxUSE_XXX,md which does describe it).

You mean `include/wx/msw/chkconf.h` right? because this is MSW only option.

> Also, it would be great if you could please update configure too, but if you really can't, I'll do it myself.

Sorry, I am not familiar with configure at all :(
Thanks

Reply:
> This `#if defined(wxUSE_IPV6) || defined(wxUSE_WINSOCK2)` is indeed wrong, I am laughing at myself now.